bagel/tests/data/README.md

```diff
@@ -15,5 +15,6 @@ Example inputs to the CLI
 | 8       | valid, based on ex2 has multiple participant_id columns            | valid, based on ex2 multiple participant_id column annotations                   | fail*  |
 | 9       | invalid, based on example 6 but contains an unannotated value for `group`            | valid, based on example 6                   | fail  |
 | 10       | valid, same as example 6           | valid, based on example 6 but contains extra `"MissingValues"` not found in the .tsv                   | pass, with warning  |
+| synthetic       | valid, has `participant` and `session` IDs corresponding to the [`synthetic` example BIDS dataset](https://github.com/bids-standard/bids-examples/tree/master/synthetic)           | valid                   | pass  |
```

Comment:
all good, but the markdown is looking sad. I'm using a [pycharm functionality for aligning this](https://www.jetbrains.com/help/pycharm/markdown.html#table), maybe there is a similar plugin for vscode?

Reply:
not for this PR though

Reply:
Ah yes, the Prettier extension can do this - I'll make sure to run it next time. I've realized also that when line wrapping is enabled even formatted tables look super wonky.